From 2905a3e48e256ecfce8a1095cb9e4d7c240ef42b Mon Sep 17 00:00:00 2001
From: Cimbali <me@cimba.li>
Date: Mon, 20 Nov 2023 13:43:31 +0000
Subject: [PATCH 1/4] Rely on package manager for dependencies, not vcpkg

---
 internal/frontend/bridge-gui/BridgeSetup.cmake | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/internal/frontend/bridge-gui/BridgeSetup.cmake b/internal/frontend/bridge-gui/BridgeSetup.cmake
index 7c6c634c..a6471667 100644
--- a/internal/frontend/bridge-gui/BridgeSetup.cmake
+++ b/internal/frontend/bridge-gui/BridgeSetup.cmake
@@ -53,14 +53,6 @@ endif()
 
 set(VCPKG_ROOT "${BRIDGE_REPO_ROOT}/extern/vcpkg")
 message(STATUS "VCPKG_ROOT is ${VCPKG_ROOT}")
-if (WIN32)
-    find_program(VCPKG_EXE "${VCPKG_ROOT}/vcpkg.exe")
-else()
-    find_program(VCPKG_EXE "${VCPKG_ROOT}/vcpkg")
-endif()
-if (NOT VCPKG_EXE)
-    message(FATAL_ERROR "vcpkg is not installed. Run build.sh (macOS/Linux) or build.ps1 (Windows) first.")
-endif()
 
 # For now we support only a single architecture for macOS (ARM64 or x86_64). We need to investigate how to build universal binaries with vcpkg.
 if (APPLE)
@@ -86,5 +78,3 @@ if  (WIN32)
     message(STATUS "Building for Intel x64 Windows computers")
     set(VCPKG_TARGET_TRIPLET x64-windows)
 endif()
-
-set(CMAKE_TOOLCHAIN_FILE "${VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake" CACHE STRING "toolchain")
\ No newline at end of file
-- 
2.42.1

